src/test/data/JsonCourseBookStorageTest/invalidAndValidPersonCourseBook.json

```diff
@@ -2,12 +2,10 @@
   "persons": [ {
```

Comment:
i think the all `.json` test data  files, should  also  have their files names changed (refactored) to e.g, `invalidAndValidModuleCourseBook.json,` etc, I'll try to change it.

Reply:
Oh no I missed this sorry!